src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.module.ts

```diff
@@ -5,10 +5,10 @@ import { CheckboxModule, ToolbarModule, ButtonModule } from 'primeng/primeng';
 import { FormsModule } from '@angular/forms';
 import { DotEditPageViewAsControllerModule } from '../dot-edit-page-view-as-controller/dot-edit-page-view-as-controller.module';
 import { DotEditPageStateControllerModule } from '../dot-edit-page-state-controller/dot-edit-page-state-controller.module';
-import { DotEditToolbarModule } from '@portlets/dot-edit-page/main/dot-edit-toolbar/dot-edit-toolbar.module';
 import { DotEditPageInfoModule } from '@portlets/dot-edit-page/components/dot-edit-page-info/dot-edit-page-info.module';
 import { DotGlobalMessageModule } from '@components/_common/dot-global-message/dot-global-message.module';
 import { DotEditPageWorkflowsActionsModule } from '../dot-edit-page-workflows-actions/dot-edit-page-workflows-actions.module';
+import {DotSecondaryToolbarModule} from '@components/dot-secondary-toolbar/dot-secondary-toolbar.module';
```

Comment:
space add a index to we can import like `@components/dot-secondary-toolbar/`

We should be doing that all the time.